Adyen/Analytics/AnalyticsProvider/AnalyticsProvider.swift

```diff
@@ -146,6 +146,15 @@ internal final class AnalyticsProvider: AnalyticsProviderProtocol {
         }
     }
     
+    private func removeSentEvents(from request: AnalyticsRequest) {
```

Comment:
What do you think about that naming? Might be a bit more clear 🤔 
```suggestion
    private func removeEvents(sentBy request: AnalyticsRequest) {
```

Reply:
I actually find `removeSentEvents` easier to understand. My opinion

Reply:
Like this suggestion!